### ISSUE TITLE
Option to configure LZip dictionary size

### ISSUE THREAD

**Issue by exyi** [adamhathcock/sharpcompress#635]
Hi! This is an awesome library btw :)

I'd like to increase the LZip dictionary size, the default seems to be only ~100 (not sure though), but I don't see any option to overwrite it.

It seems that it's only controlled in this code: https://github.com/adamhathcock/sharpcompress/blob/master/src/SharpCompress/Compressors/LZMA/LZipStream.cs#L43, so it seems reasonably simple to implement. Would you accept a PR for another `new LZipStream()` overload which would allow setting this parameter.

**Comment by adamhathcock** [comment]
Of course.   Reasonable changes are always excepted.  Any change can be discussed!
